examples/js/loaders/GLTFLoader.js

```diff
@@ -261,6 +276,23 @@ THREE.GLTFLoader = ( function () {
 							extensions[ extensionName ] = new GLTFMeshQuantizationExtension();
 							break;
 
+						case EXTENSIONS.EXT_MESHOPT_COMPRESSION:
+							if ( extensionsRequired.includes( extensionName ) ) {
+
+								if ( !this.meshoptDecoder ) {
```

Comment:
nit: space between `! this.meshoptDecoder` here and below, I think the linter will complain otherwise.